src/Form/Field/BlocksField.php

```diff
@@ -238,19 +239,6 @@ public function routes(): array
 		];
 	}
 
-	public function store(mixed $value): mixed
```

Comment:
`Question:` Deprecate before removal?

Reply:
If we want to keep this around and deprecate it, we would need to add the string conversion code back in there and use it in the `::toStoredValue` method or add the same code twice.